test.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/env bash
 set -e
 
+export FOUNDRY_ROOT_CHAINID="$(cast chain-id)"
+if [ "$FOUNDRY_ROOT_CHAINID" != "1" ] && [ "$FOUNDRY_ROOT_CHAINID" != "5" ]; then
+  echo "Invalid chainid of $FOUNDRY_ROOT_CHAINID. Please set your forking environment via ETH_RPC_URL."
+  exit
+fi
+
```

Comment:
I think I like our historical check for this as it is a little more clear than letting cast fail and then throwing another error

```suggestion
[[ $ETH_RPC_URL  ]] || { echo "Please set an ETH_RPC_URL"; exit 1; }

export FOUNDRY_ROOT_CHAINID="$(cast chain-id)"
if [ "$FOUNDRY_ROOT_CHAINID" != "1" ] && [ "$FOUNDRY_ROOT_CHAINID" != "5" ]; then
  echo "Invalid chainid of $FOUNDRY_ROOT_CHAINID. Please set your forking environment via ETH_RPC_URL."
  exit
fi
```